lms/djangoapps/verify_student/models.py

```diff
@@ -952,6 +952,11 @@ class VerificationDeadline(TimeStampedModel):
         )
     )
 
+    # The system prefers to set this automatically based on default settings. But
+    # if the field is set manually we want a way to indicate that so we don't
+    # overwrite the manual setting of the field.
+    is_override_deadline = models.BooleanField(default=False)
```

Comment:
The migration should probably set all existing dates to `True`, even thought the ongoing default will be `False`.